JS-Algorithms/challenges.js

```diff
@@ -95,7 +133,19 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 ***** */
 
 const digitSum = (n) => {
-  // YOUR CODE HERE...
+  let factorialOfN = BigInt(1);
+
+  for (let i = BigInt(n); i > 1; i--) {
+    factorialOfN *= i;
+  }
+
+  const digitsInfactorialOfN = [...factorialOfN.toString()].map(
```

Comment:
Because `factorialOfN.toString()` returns a string, using the spread operator splits the string into individual characters and place them in an array, allowing me to apply a `map`.